### PR TITLE
Makes more stuff destructible/breakable

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

Draft until I get to the rest of the stuff cause I wanna play Enigmatica for a bit.

Microwave might be jank just because of how convoluted it's code is. I kinda wish we had a global "broken" visualizer to go along with the [Breakable] DoActsBehavior but this works for now.

I increased the hit-points on lockers and secure lockers and you can check the commit log for the rest of the stuff. ~~I plan to get to substations, piano breakable state, dispensers, APC, hydroponics computer, morgue and cell recharger.~~ Saved for another PR.

**Changelog**

:cl: Swept
- tweak: More machines and station equipment is destructible

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
Ready for review. Tested on my end after pulling master and looks fine. Build failing cause this PR is ancient. Shouldn't have left this so long but just got bored and sidetracked.

File: Content.Client/Kitchen/Visualizers/MicrowaveVisualizer.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 using Robust.Shared.Audio;
 using Robust.Shared.Log;
 
-namespace Content.Client.Kitchen.Visualizers
+namespace Content.Client.GameObjects.Components.Kitchen
```
Comment (mirrorcult) [inline Content.Client/Kitchen/Visualizers/MicrowaveVisualizer.cs:10]: incorrect namespace

File: Resources/Prototypes/Entities/Constructible/Power/seed_extractor.yml [hunk 2]
```diff
@@ -5,24 +5,43 @@
...
-        layer:
-          - Opaque
-          - Impassable
-          - MobImpassable
-          - VaultImpassable
-    - type: SnapGrid
-    - type: Anchorable
-    - type: SeedExtractor
-    - type: PowerReceiver
+  - type: Clickable
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Constructible/Power/seed_extractor.yml:8]: can this prototype use BaseMachinePowered?

**Comment by SweptWasTaken** [comment]
Reviews applied. I added a destruction sound to BaseMachine in the process since 99% are gonna be made out of metal anyway.

**Comment by SweptWasTaken** [comment]
Self-merging in an hour if nobody objects.

**Review by Acruid — APPROVED** [review]

**Pull request merged**